### DISCUSSION TITLE
Is there no way to create snapshots like @2006-01-02-150405Z

### DISCUSSION THREAD

**Discussion by skirmess** [zrepl/zrepl#690] [Q&A]
Is there no way to create snapshots like @2006-01-02-150405Z? When I leave the prefix empty I get the following error message on startup:

> field prefix is required but not given

Is there a workaround for this?

**Comment by InsanePrawn** [comment]
Does an empty string (`prefix: ""`) work?

Otherwise we should probably allow explicitly setting it to an empty string.

**Comment by skirmess** [comment]
No, `prefix: ""` doesn't work

```
could not parse config: yaml: unmarshal errors:
  line 16: field prefix is required but has zero value
  line 1: field snapshotting is required but has zero value
  line 0: field jobs is required but has zero value
```

**Comment by skirmess** [comment]
I've created https://github.com/zrepl/zrepl/issues/694 to track the request.
